product/api.go

```diff
@@ -80,7 +80,7 @@ type LicenseService interface {
 // The service shall be registered via app.UserKey service key.
 type UserService interface {
 	GetUser(userID string) (*model.User, *model.AppError)
-	UpdateUser(user *model.User, sendNotifications bool) (*model.User, *model.AppError)
+	UpdateUser(c request.CTX, user *model.User, sendNotifications bool) (*model.User, *model.AppError)
```

Comment:
@isacikgoz - Doug is on PTO, so wanted to confirm: will this require any changes on the boards side? From a brief look at the code, it looks like it doesn't. But wanted to check with you.

Reply:
This may require change on their end as they use this service API AFAIK. I will check once I get back home.

Reply:
@agnivade just checked and I can confirm that this requires changes on board side. Thanks for pointing out this actually escaped from me. Since we are here, I think we should also at least build focalboard as part of the CI process.

Reply:
I thought @wiggin77 already did that? Not sure if there were any steps pending for that. Can you send the PR that makes the changes on board side? Let's merge all 3 together.

Reply:
Yeah sure 👍